src/views/DashboardView.vue

```diff
@@ -1,3 +1,36 @@
+<script setup>
+import ContributionTable from "../components/ContributionTable.vue";
+import { ref, watchEffect } from "vue";
+import { getLatestContributions, getMyContributions } from "../stores/api.js";
+
+function calcPercentMissing(numberMissing, numberTotal) {
+  return ((numberMissing / numberTotal) * 100).toFixed(2);
+}
+
+const userName = ref("Hannah Waruguru");
+const userContributions = ref(null);
+watchEffect(async () => {
+  userContributions.value = await getMyContributions(userName.value)
```

Comment:
Let's add a try catch block here for error handling

Reply:
Hi, @Damilare1  since the Dashboard has multiple tables with calls to different endpoints, I am struggling with ideas in case there is one or several of the table's data missing.

Reply:
Hmm, let's discuss this on our call.